jekyll/_docs/configuration.md

```diff
@@ -512,9 +510,9 @@ deployment:
       - ./deploy_master.sh
 ```
 
-***Note*** The `deployment` section doesn't support sub-sections such as `pre` 
-that you might find elsewhere in `circle.yml`. If you get a strange error such 
-as "commands must be a list" when in the `deployment` section, this is likely 
+***Note*** The `deployment` section doesn't support sub-sections such as `pre`
+that you might find elsewhere in `circle.yml`. If you get a strange error such
+as "commands must be a list" when in the `deployment` section, this is likely
```

Comment:
Thanks for doc updates!

Reading this... Might Circle get a better about giving user friendly errors for circle.yml problems? Also, it'd be awesome to publish that linter so that users can rely on it and run it locally